### DISCUSSION TITLE
NPC Aggression Timer notification

### DISCUSSION THREAD

**Discussion by ktilk** [runelite/runelite#15647] [Ideas]
Suggestion to add notification when NPC aggression expires AND player is out of combat.

**Comment by Enriath** [comment]
This would functionally be not that different from just notifying on exiting combat (which is currently possible with `Idle Notifier`)

**Reply by ktilk** [reply]
Proposed feature would make notifications less spammy. For example it would not trigger notification when waiting for and aggressive NPC to respawn.
